### DISCUSSION TITLE
How to compress an image by the trained model

### DISCUSSION THREAD

**Discussion by wojiaobaixiaofei** [tensorflow/compression#166] [Q&A] [answered]
Dear authors,
  I want to know how to compress an image by the trained model?

**Comment by jonaballe** [comment, marked as answer]
Have you tried to follow https://github.com/tensorflow/compression#using-a-pre-trained-model-to-compress-an-image?

**Reply by wojiaobaixiaofei** [reply]
Thank you for your answer. I try it.
